docs/integrations/app-development/opentelemetry/jira.md

```diff
@@ -0,0 +1,225 @@
+---
+id: jira-opentelemetry
+title: Jira - OpenTelemetry Collector
+sidebar_label: Jira - OTel Collector
+description: The Sumo Logic App for Jira provides insight into Jira user access, request activity, issues, security, sprint events, and user events.
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<img src={useBaseUrl('img/integrations/app-development/jira.png')} alt="Thumbnail icon" width="50"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
```

Comment:
This image link is broken when I run it in a local build. Verify that it works in your system.